File: assignment2_q1a.c
#include<stdio.h>
#include<stdlib.h>
int main(){
	

	int no_points = 6;
	float pairs = 6*6;

	float total = 0;
	for (int i = 0; i < no_points; ++i){
		for (int j = 0; j < no_points; ++j){
			total = total + abs(i-j);
		}
	}

	float avg = total/pairs;

	printf("The average distance is ");
	printf("%.4f",avg);

}

File: assignment2_q1a.py
points = []

#  Assign co-ordinates
for n in range(6):
	points.append(n)

total = 0
pairs = 6**2

for i in points:
	for j in points:
		total += abs(j - i)

avg = total/pairs

print("The average distance is ", avg)